spec/controllers/todos_controller_spec.rb

```diff
@@ -126,4 +126,100 @@
       end
     end
   end
+
+  describe '#update' do
+    subject { patch :update, params: }
+
+    let(:params) { default_params }
+    let(:default_params) do
```

Comment:
Why do we need this alias ?

Reply:
805fad88f45edfc2709c9382bf82bd125265c227